charts/base-services/templates/rules/kubernetes-storage.yaml

```diff
@@ -33,7 +33,7 @@ spec:
         runbook_url: https://github.com/kubernetes-monitoring/kubernetes-mixin/tree/master/runbook.md#alert-name-kubepersistentvolumefullinfourdays
       expr: |
         kubelet_volume_stats_available_bytes{job="kubelet"} and predict_linear(kubelet_volume_stats_available_bytes{job="kubelet"}[6h], 8 * 24 * 3600) < 0
```

Comment:
Just to double check: based on a history of 6 hours, we are linear predicting 8 days forward

Reply:
Yes exactly, when the predicted space left is less than 0 for 30 min we were triggering the alert. This gives us a lot of false positives when there are spikes of usage, maybe bc an application is flushing temporary data or moving files around. Increasing the `for` field can prevent this kind of problem. 

We could also extend the 6h used for the prediction to average over a wider window, any suggestions are more than welcome :)